modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp

```diff
@@ -165,7 +167,10 @@ BlogsPortletInstanceConfiguration blogsPortletInstanceConfiguration = BlogsPortl
 
 		<!-- text resume -->
 
-		<div class="container widget-mode-detail-header" id="<portlet:namespace /><%= entry.getEntryId() %>">
+		<clay:container
+			className="widget-mode-detail-header"
+			id="<portlet:namespace /><%= entry.getEntryId() %>"
```

Comment:
`id='<%= renderResponse.getNamespace() + entry.getEntryId() %>'` ?

Reply:
yes please, in this case, since there's no double quotes inside, you can use them outside:

`id="..."`